testutils/src/jest-shim.ts

```diff
@@ -52,3 +52,12 @@ process.on('unhandledRejection', (error, promise) => {
   }
   promise.catch(err => console.error('promise rejected', err));
 });
+
+(window as any).getSelection = function getSelection() {
```

Comment:
Should this check if `window.getSelection()` already exists or is it safe to do what you've done here?

Reply:
Given the limited scope we're using it for, I think this is fine.  Interestingly, newer versions of jsdom do have this method though.